dbt/models/census/columns.md

```diff
@@ -0,0 +1,757 @@
+
+{% docs column_median_household_renter_occupied_gross_rent %}
+Median gross rent for only renter-occupied units.
+
+ACS variable `B25064_001E`
+{% enddocs %}
+
+## median_household_renter_occupied_year_built
+
+{% docs column_median_household_renter_occupied_year_built %}
+Median year built for only renter-occupied units.
+
+ACS variable `B25037_003E`
+{% enddocs %}
+
+## median_household_total_occupied_year_built
+
+{% docs column_median_household_total_occupied_year_built %}
+Median year built for all occupied households.
+
+ACS variable `B25037_001E`
+{% enddocs %}
+
+## median_income_household_past_year
+
+{% docs column_median_income_household_past_year %}
+Median income per household in the past 12 months.
+
+ACS variable `B19013_001E`
+{% enddocs %}
+
+## median_income_per_capita_past_year
+
+{% docs column_median_income_per_capita_past_year %}
+Median income per capita in the past 12 months.
+
+ACS variable `B19301_001E`
+{% enddocs %}
+
+## percent_age_children
+
+{% docs column_percent_age_children %}
+Percent of the people 17 years or younger.
+
+ACS variable (`B01001_003E` + `B01001_004E` + `B01001_005E` +
+`B01001_006E` + `B01001_007E` + `B01001_027E` + `B01001_028E` +
+`B01001_029E` + `B01001_030E` + `B01001_031E`) / `B01001_001E`
+{% enddocs %}
+
+## percent_age_senior
+
+{% docs column_percent_age_senior %}
+Percent of the people 65 years or older.
+
+ACS variable (`B01001_020E` + `B01001_021E` + `B01001_022E` +
+`B01001_023E` + `B01001_024E` + `B01001_025E` + `B01001_044E` +
+`B01001_045E` + `B01001_046E` + `B01001_046E` + `B01001_048E` +
+`B01001_049E`) / `B01001_001E`
+{% enddocs %}
+
+## percent_education_associate
+
+{% docs column_percent_education_associate %}
+Percent of people older than 25 who attained an associate's degree.
+
+ACS variable (`B15002_014E` + `B15002_031E`) / `B15002_001E`
+{% enddocs %}
+
+## percent_education_bachelor
+
+{% docs column_percent_education_bachelor %}
+Percent of people older than 25 who attained a bachelor degree.
+
+ACS variable (`B15002_015E` + `B15002_032E`) / `B15002_001E`
+{% enddocs %}
+
+## percent_education_graduate
+
+{% docs column_percent_education_graduate %}
+Percent of people older than 25 who attained a graduate degree.
+
+ACS variable (`B15002_016E` + `B15002_017E` + `B15002_018E` +
+`B15002_033E` + `B15002_034E` + `B15002_035E`) / `B15002_001E`
+{% enddocs %}
+
+## percent_education_high_school
+
+{% docs column_percent_education_high_school %}
+Percent of people older than 25 who attained a high school degree.
+
+ACS variable (`B15002_011E` + `B15002_028E`) / `B15002_001E`
+{% enddocs %}
+
+## percent_employment_unemployed
+
+{% docs column_percent_employment_unemployed %}
+Percent of people 16 years and older unemployed.
+
+ACS variable `B23025_005E` / `B23025_003E`
+{% enddocs %}
+
+## percent_household_family_married
+
+{% docs column_percent_household_family_married %}
+Percent of households that are family, married (married).
```

Comment:
**[Question, non-blocking]** What's the distinction between the two `married`s here?

Reply:
Woops, this is a typo!